data.sql

```diff
@@ -76,4 +76,10 @@ VALUES
 (2, 7, '2019/05/15'),
 (2, 7, '2020/02/27'),
 (3, 7, '2020/05/24'),
-(1, 10, '2021/01/11');
\ No newline at end of file
+(1, 10, '2021/01/11');
+
+
+/* New data */
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
- [ ] Good job adding the Data Team, the problem is the execution time in all the screenshots BEFORE is significantly lower than 1 sec (1000ms). I would suggest running the populate database query a few times more as described int he [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.). We need to see more data being inserted so that your execution takes more time before, this enables us to see how much the performance is improved.